index.rst

```diff
@@ -88,6 +88,7 @@ networking and security configuration.
    Network Options <networking>
    Limiting Container Resources <cgroups>
    Application Checkpointing <checkpoint>
+   Monitoring <monitoring>
```

Comment:
I believe the new monitoring page belongs in the admin guide instead of the user guide.

Reply:
yeah thanks! moved to admindocs.

Reply:
https://github.com/apptainer/apptainer-admindocs/pull/145